lib/jsonapi/filtering.rb

```diff
@@ -62,7 +64,14 @@ def jsonapi_filter_params(allowed_fields)
           to_filter = to_filter.split(',')
         end
 
-        if predicates.any? && (field_names - allowed_fields).empty?
+        # Disable predicate validation for scopes if the scope name is allowed
+        allow = predicates.any? || (
+          predicates.none? &&
+          scopes.include?(requested_field) &&
+          allowed_fields.include?(requested_field)
```

Comment:
Hm, given that we handle scopes separately already, I don't really see why we should add the scopes to the `allowed_fields` array as well. Most scopes will have names that don't directly match the name of a field, so I think this would be a little confusing. 

However, I might be missing something as well!

Reply:
> Most scopes will have names that don't directly match the name of a field, so I think this would be a little confusing.

Yup, I was thinking about it too and the only reason I'd leave it explicit, is because there might be scopes that can conflict with the field name. My concern is that we might accidentally enable a scope as filterable without the user being aware of it.

@fluxsaas would be nice to get your opinion on this too. :bow: